storybook/stories/Login/index.tsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+import { storiesOf } from "@storybook/react-native";
+import { View } from "react-native";
+import { size } from "../../../src/common/styles";
+import { LoginMobileNumberCard } from "../../../src/components/Login/LoginMobileNumberCard";
+import { LoginOTPCard } from "../../../src/components/Login/LoginOTPCard";
+import { LoginScanCard } from "../../../src/components/Login/LoginScanCard";
+import { AppText } from "../../../src/components/Layout/AppText";
+
+storiesOf("Login", module)
+  .add("MobileNumberCard", () => (
+    <View style={{ margin: size(3) }} key="0">
```

Comment:
Why do we need the `key`?

Reply:
this will solve the warning by react-dom. without `key` we will got a warning message like this
<img width="413" alt="Screenshot 2021-04-28 at 10 30 43 AM" src="https://user-images.githubusercontent.com/66373739/116337562-e6c9f080-a80c-11eb-89d6-30e34329c2ed.png">